### PR TITLE
chore(l10n): Update Polish translation

### PR DESCRIPTION
**Pull request by carno**
Ref: https://github.com/newsboat/newsboat/pull/2282

### CODE REVIEW

File: po/pl.po [hunk 1]
```diff
@@ -517,13 +519,13 @@ msgid "Exporting list of read articles..."
...
 #: src/controller.cpp:572
-#, fuzzy, c-format
+#, c-format
 msgid ""
 "%<PRIu64> unreachable feeds found. See `cleanup-on-quit` in newsboat(1) for "
 "details."
 msgstr ""
-"Znaleziono nieosiągalne kanały, rozważ ustawienie `cleanup-on-quit yes` lub "
-"uruchom `newsboat --cleanup`"
+"%<PRIu64> nieosiągalnych kanałów. Więcej informacji pod  `cleanup-on-quit` w "
```
Comment (der-lyse) [inline po/pl.po:527]: I know nothing about Polish. There are two spaces in between `pod` and ````cleanup-on-quit````. Is this intentional or should it be just one?

**Review by der-lyse — APPROVED** [review]
Maybe one thing. Otherwise looking good.

File: po/pl.po (hunk 1 above)
Comment (carno) [reply po/pl.po]: Nice catch. No, it was not intentional, should be fixed now.

**Comment by Minoru** [comment]
Thanks!

**Pull request merged**